cli/azd/pkg/project/project_config.go

```diff
@@ -37,7 +37,9 @@ type RequiredVersions struct {
 
 // options supported in azure.yaml
 type PipelineOptions struct {
-	Provider string `yaml:"provider"`
+	Provider  string   `yaml:"provider"`
+	Variables []string `yaml:"variables"`
+	Secrets   []string `yaml:"secrets"`
```

Comment:
Part of me feels like we are setting a bad example saying that we are expecting application secrets to be available in the local environment when in reality your azd environment should not have any secrets or passwords but instead reference a secure secret store like keyvault.

Reply:
conversation started about this internally.